src/use-swr.ts

```diff
@@ -678,9 +678,25 @@ function useSWR<Data = any, Error = any>(
 Object.defineProperty(SWRConfigContext.Provider, 'default', {
   value: defaultConfig
 })
-const SWRConfig = SWRConfigContext.Provider as typeof SWRConfigContext.Provider & {
+
+export const SWRConfig = SWRConfigContext.Provider as typeof SWRConfigContext.Provider & {
   default: SWRConfiguration
 }
+export const mutate = internalMutate.bind(
+  null,
+  defaultConfig.cache
+) as ScopedMutator<any>
+export function createCache<Data>(
+  provider: Cache
+): {
+  cache: Cache
+  mutate: ScopedMutator<Data>
+} {
+  const cache = wrapCache<Data>(provider)
+  return {
+    cache,
+    mutate: internalMutate.bind(null, cache) as ScopedMutator<Data>
+  }
+}
```

Comment:
is deprecating `trigger` also be included in this PR's scope?

Reply:
Yeah I think we have to, because it should be scoped to the cache just like `mutate`, so it's better to just deprecate it.